src/runtime/opencl/opencl_device_api.cc

```diff
@@ -273,9 +270,6 @@ void OpenCLWorkspace::Init(const std::string& type_key, const std::string& devic
     this->queues.push_back(
         clCreateCommandQueue(this->context, did, 0, &err_code));
     OPENCL_CHECK_ERROR(err_code);
-    LOG(INFO) << type_key << "(" << i
-              << ")=\'" << cl::GetDeviceInfo(did, CL_DEVICE_NAME)
-              << "\' cl_device_id=" << did;
   }
 }
```

Comment:
Are these removed because it pollutes the log? You shouldn't remove them unless there is a good reason.

Reply:
Yes it pollutes the log.
Should we keep them? Because we don't do this for cuda and cuda works well. I think we can query this information by clinfo.

Reply:
how about LOG(INFO) -> LOG(DEBUG)?